### PR TITLE
Reduce tumbleweed size (it's ~ 99 MB now)

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: tests/test_base.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def test_passwd_present(auto_container):
 
 @pytest.mark.skipif(
     OS_VERSION not in ("tumbleweed",),
-    reason="requires glibc-locale-base installed",
+    reason="requires gconv modules installed",
```
Comment (dcermak) [inline tests/test_base.py:40]:
```suggestion
    reason="requires gconv modules to be installed",
```

**Review by dcermak — APPROVED** [review]
LGTM, but maybe rebase to be safe?

**Review by alexandrevicenzi — DISMISSED** [review]

File: tests/test_base.py (hunk 1 above)
Comment (dirkmueller) [reply tests/test_base.py]: I wanted shorter, not longer. I've removed installed now.

**Comment by dirkmueller** [comment]
> LGTM, but maybe rebase to be safe?

it has two approvals already...

**Pull request merged**